cime_config/acme/machines/template.case.run

```diff
@@ -112,7 +113,8 @@ def preRunCheck(case):
     preview_namelists()
 
     # document process
-    appendCaseStatus(caseroot, "%s Run started \n" %(time.strftime("%Y-%m-%d %H:%M:%S")))
```

Comment:
I reworked this to allow appending any file and use it for all of CaseStatus, TestStatus and TestStatus.log
CaseStatus and TestStatus.log get a timestamp prepended to each entry.